rules/python/django/path_using_user_input.yml

```diff
@@ -2,8 +2,12 @@ imports:
   - python_shared_common_user_input
   - python_shared_lang_import4
 patterns:
-  - pattern: $<FILE_SYSTEM_STORAGE>($<...>$<USER_INPUT>$<...>)
+  - pattern: $<FILE_SYSTEM_STORAGE>($<...>$<PARAM>=$<USER_INPUT>$<...>)
```

Comment:
What position are these args in? Should we support positional args as well?

Reply:
Ahh thanks, yes, we definitely should support positional args 🙏  
I've added the patterns for this now (for reference, location and base url are [1st and 2nd params - see docs](https://docs.djangoproject.com/en/5.0/ref/files/storage/#the-filesystemstorage-class))